CMakeLists.txt

```diff
@@ -79,6 +79,34 @@ find_package(ZLIB REQUIRED)
 
 
 
+
```

Comment:
You can use the newer version of piemaker to generate the cmakelist and you will get a more compacted version of cmake list file. Probably we should avoid checking this file into version control later.

Reply:
Since this is file is used when vcpkg install, if we don't check into this file into version control, how will vcpkg get the latest CMakeLists.txt generated by piemaker?